File: C++/program4.cpp
// Accept values from user then calculate percentage 

#include<iostream>
using namespace std;

float CalculatePercentage(int total, int obtained)
{
    float percentage = 0.0f;
    percentage = ((float) obtained / (float) total )* 100;

    return percentage;

}

int main()
{
    int Total = 0;
    int Obtained = 0;
    float fRet = 0.0f;

    cout<<"Enter total marks  "<<'\n';
    cin>>Total;

    cout<<"Enter obtained marks "<<'\n';
    cin>>Obtained;

    fRet = CalculatePercentage(Total,Obtained);

    cout<<"Percentage is :"<<fRet<<'\n';
    return 0;
}

File: Java/program4.java
// Calculate perecntage 

import java.util.*;

class program4
{
    public static void main(String A[])
    {
        int Total = 0, Obtained = 0;
        float perecntage = 0.0f;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter your total marks :");
        Total = sobj.nextInt();

        System.out.println("Enter your obtained marks :");
        Obtained = sobj.nextInt();

        perecntage = ((float)Obtained / (float)Total) * 100;
        System.out.println("Percentage is :"+perecntage);
        
    }
}